missilewars-plugin/src/main/java/de/butzlabben/missilewars/wrapper/game/Team.java

```diff
@@ -233,7 +226,7 @@ public void sendGameResultSound(MWPlayer missileWarsPlayer) {
         }
     }
 
-    // TODO Review for necessity (of an alternative), completeness and refactoring
+    // TODO Add new team buffer
```

Comment:
What new team buff?

Reply:
In the case that too many players leave a team in the lobby. I will look at this in the future and check what is necessary there.

Reply:
Then please change to comment something like:
`// TODO Maybe add more new team buffs in case more players leave and the get's more unbalanced`